changelogs/fragments/442-log_plays-add_playbook_task_name_and_action.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+- log_plays - Use v2 methods.
+- log_plays - Add missing information to the logs generated by the log_plays callback.
```

In case someone depends on the precise log message format, I'd mark this as a breaking change:
```suggestion
breaking_changes:
- log_plays callback - add missing information to the logs generated by the callback plugin. This changes the log message format. (https://github.com/ansible-collections/community.general/pull/442)
```